mirgecom/heat.py

```diff
@@ -0,0 +1,115 @@
+r""":mod:`mirgecom.heat` computes the rhs of the heat equation.
```

Comment:
Replace `heat` with `diffusion` (everywhere) to more closely match the role that this will play in Navier-Stokes.